package.json

```diff
@@ -70,14 +71,12 @@
     ],
     "all": true
   },
-  "peerDependencies": {
-    "semantic-release": ">=18.0.0"
-  },
   "prettier": {
     "printWidth": 120,
-    "trailingComma": "es5",
-    "singleQuote": true,
-    "bracketSpacing": false
+    "trailingComma": "es5"
+  },
+  "peerDependencies": {
+    "semantic-release": ">=18.0.0"
```

Comment:
since this version of the gitlab plugin will require a version of semantic-release that is capable of loading ESM plugins, i would recommend setting this to `>=v20.1.0`